conf.py

```diff
@@ -98,7 +98,7 @@
 
 args = {
     "STACKABLE_USER_NAME": "stackable",
-    "STACKABLE_USER_UID": "1000",
-    "STACKABLE_USER_GID": "1000",
+    "STACKABLE_USER_UID": "782252253",
```

Comment:
Can you please commented in code how you got these numbers?
Picked at random? If so please add a comment or link to best practices

Reply:
I can add a comment, yes.
Just like the previous ones these are random, yes.

Reply:
[`bf5a6a4` (#916)](https://github.com/stackabletech/docker-images/pull/916/commits/bf5a6a4cf3cd8ec53bb5281a15dd5cbcb4e80a6a)

Reply:
Thanks!